src/tools/dataOrganizer/drawer.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const fs = require('fs');
+
+class Drawer {
+  static SYMBOLS = {
+    corners: { topLeft: '┌', topRight: '┐', bottomLeft: '└', bottomRight: '┘' },
+    lines: { horizontal: '─', vertical: '│' },
+    ternaryLines: { top: '┬', left: '├', right: '┤', bottom: '┴' },
+    quadroLine: '┼',
+  };
+  constructor(obj, mode, settings = {}) {
+    this.mode = mode;
+    this.settings = settings;
+    this.stringImage = this._getImage(obj);
+  }
+
+  _getImage(obj) {
+    let res = [];
+    if (this.mode === 'table' || !this.mode) {
+      res = this._drawTable(obj);
+    } else {
+      throw new Error('Unexpected mode key!');
+    }
+    return res.join('');
+  }
+
+  _drawTable(table) {
+    console.log(table);
+    const res = [];
+    const obj = table.obj;
+    const cols = table.colsNum;
+    const horOffset = table.maxValueLength;
+
+    res.push(this._createTableLine('top', cols, horOffset));
+    Object.keys(obj).forEach((key, i) => {
+      const raw = [];
+      const values = [key].concat(
+        Array.isArray(obj[key]) ? obj[key] : [obj[key]]
+      );
+
+      raw.push(Drawer.SYMBOLS.lines.vertical);
+      values.forEach(value => {
+        value = String(value);
```

Comment:
Changing the argument is not good.. Maybe create a new variable called stringValue or smth